datasource/models/wikidata/wikidata.py

```diff
@@ -72,7 +68,7 @@ def only_allowed(val):
                 traceback.print_exc()
 
         def add_subsidiary(bank, parent):
```

Comment:
Won't this break? Subsidiary information remains in the Brand model but now wikidata doesn't inherit from brand. I'd suggest either commenting this out to deal with it later or building a binary subsidiary model into wikidata. (i.e. `subsidiary_of=models.ForeignKey(Wikidata...)`

Reply:
yes this would break; I'd removed the code that calls it but now also commented this out.